source/administration/incident-response-application.rst

```diff
@@ -0,0 +1,667 @@
+                          "description": "Issue has been triaged",
+                          "to": "DevelopFix"
+                      },
+                      {
+                          "label": "Resolved",
+                          "description": "Issue has been resolved",
+                          "to": "Resolved"
+                      }
+                  ]
+              }
+          ]
+      }
+  ]
+
+.. tip::
+  Note that the above JSON uses ``{{.Action.CreateWarroom.ChannelName}}`` as the channel name. These are template variables which allow you to dynamically specify parameters based on other actions or steps within the app workflow schema.
+  
+  In this example, ``{{.Action.CreateWarroom.ChannelName}}`` pulls the channel name used in an ``CreateWarroom`` action, which is the example of the ``create_channel`` action above, and posts a message to that channel.
+
+**Fields and Transitions**
+
+Note that in the above JSON example, the post action type contained fields and transitions.
+
+Fields are usually dropdown menus or buttons that allow users to add more details to the incident, such as escalate an issue to the Team Lead, and specify the impact and reach of the incident.
+
+You can also optionally set a title for them - if none specified, ``Fields`` is used.
+
+.. image:: ../images/incident-response-app-fields.png
+   :alt: Incident Response App: Fields
+
+Transitions allow a user to move the app workflow to the next step in the process, such as **Triage** or **Resolved**. When a user clicks on one of the transitions, the workflow transitions to the specified step.
+
+You can also optionally set a title for them - if none specified, ``Transitions`` is used.
+
+.. image:: ../images/incident-response-app-transitions.png
+   :alt: Incident Response App: Transitions
+
+Transition to Another Step (type: ``transition_to``)
+******************************************************
+
+Specifies which step to transition the workflow to.
+
+.. csv-table::
+    :header: "Field", "Description", "Type", "Required"
+
+    "to", "The name of the target step to transition to", "string", "Yes"
+
+Below is an example JSON for a step containing one ``transition_to`` start action, which transitions the workflow to ``Triage`` step.
+
+.. code-block:: json
+
+  "steps": [
+      {
+          "name": "Setup",
+          "start_actions": [
+              {
+                  "name": "TransitionToTriage",
+                  "type": "transition_to",
+                  "to": "Triage"
+              },
+          ]
+      }
+  ]
+
+Archive Channel (type: ``archive_channel``)
+**********************************************
+
+Archives the specified channel.
+
+.. csv-table::
+    :header: "Field", "Description", "Type", "Required"
+
+    "channel_name", "The channel to archive.", "string", "Yes"
+    "team_name", "The team the channel belongs to. Use it if the same channel name exists in different teams.", "string", "No"
```

Comment:
@crspeller Added `team_name` but from https://github.com/mattermost/mattermost-plugin-workflow/blob/master/server/action/archivechannel.go#L15 looks like it's not required. Confirming this is expected and not a bug (you mentioned [here](https://github.com/mattermost/docs/pull/3047#discussion_r337180463) it might be required)?